src/js/components/organisms-page/resellers/resellers.js

```diff
@@ -32,6 +43,41 @@ export default {
         },
         showAllAuthorizedResellers() {
             this.visibleAllAuthorizedResellers = true;
+        },
+        newRow(el) {
+            const rect = el.getBoundingClientRect(),
+                index = el.dataset.index - this.showMax;
+
+            cardsYPos.push(rect.top);
```

Comment:
I think you only need to store the last elements top value, this way you can avoid storing all the value's in an array.